src/Roslyn.Diagnostics.Analyzers/Core/ImportingConstructorShouldBeObsoleteCodeFixProvider.cs

```diff
@@ -206,7 +207,7 @@ private static SyntaxNode GenerateDescriptionArgument(SyntaxGenerator generator,
                 mefConstructionType.GetMembers("ImportingConstructorMessage").OfType<IFieldSymbol>().Any())
             {
                 attributeArgument = generator.MemberAccessExpression(
-                    generator.TypeExpressionForStaticMemberAccess(mefConstructionType),
+                    generator.TypeExpression(mefConstructionType).WithAddImportsAnnotation(),
```

Comment:
📝 This is not a correct syntax tree, but AddImportsAnnotation only works for direct type references.